src/Theme/context.ts

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { SaleorThemeContext } from './types';
+
+export const ThemeContext = React.createContext<SaleorThemeContext>({
```

Comment:
Use empty initial value to get rid off empty functions, the `useTheme` hook should check for `undefined` and throw Error so it will be type-safe.
```suggestion
export const ThemeContext = React.createContext<SaleorThemeContext | undefined>(undefined);
```